### PR TITLE
Add note about bucket permissions in PAPI v2

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -235,6 +235,12 @@ In addition, the following changes are to be expected:
 * Error messages for failed jobs might differ from V1
 * The Pipelines API log file content might differ from V1
 
+**Permissions***:
```
Comment (cjllanwarne) [inline CHANGELOG.md]: I don't like the idea of this being "documented" by adding it onto in a months (years?) old entry in the CHANGELOG. IMO this belongs in user-facing documentation which the CHANGELOG could link out to if it chose to.

... TL;DR: Can we put this in the readthedocs PAPIv2 page?

Comment (cjllanwarne) [reply CHANGELOG.md]: @ruchim this doc is talking about PAPIv2, and seems like an appropriate place for this content: https://cromwell.readthedocs.io/en/develop/backends/Google/

Then the CHANGELOG could link out to that page (or even the `.MD` file in the `docs/` directory which feeds it?)

**Comment by gemmalam** [comment]
@ruchim could you take a look at this?

**Comment by ruchim** [comment]
@cjllanwarne Just need to update the piece around private dockerhub support --otherwise approved 👍

**Pull request merged**